dist/image-sequencer-ui.js

```diff
@@ -510,7 +510,7 @@ function DefaultHtmlStepUi(_sequencer, options) {
     }
     $(step.ui.querySelector(".toggle")).on("click", () => {
       $(step.ui.querySelector('.toggleIcon')).toggleClass('fa-caret-up').toggleClass('fa-caret-down');
-      $(step.ui.querySelectorAll(".cal")).toggleClass("collapse");
+      $(step.ui.querySelectorAll(".cal")).collapse('toggle');
```

Comment:
Change the selector from '.cal' to '.step', that was the problem

Reply:
> Change the selector from '.cal' to '.step', that was the problem

Doing that will collapse the whole step including the toggle button.

Reply:
Why doesn't the image collapse then?